### PR TITLE
reflect preferred installation and changelog update

### PR DESCRIPTION
**Pull request by psytester**
In #27 you proposed the usage of pip installation instead of aptitute package manager
and changelog should be up-to-date, too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +21,16 @@ Make sure you are using the lastest version of Raspbian:
     $ sudo apt-get update
     $ sudo apt-get upgrade
 
-Install `pifacecommon` (for Python 3 and 2) with the following command:
+Install `pifacecommon` with the following commands:
 
-    $ sudo apt-get install python{,3}-pifacecommon
+    Python 2:
+    $ sudo pip pifacecommon
```
Comment (tompreston) [inline README.md:27]: I'd remove this, Python 2 was sunset Jan 1 2020 and should no longer be used. https://www.python.org/doc/sunset-python-2/

File: README.md [hunk 2]
```diff
@@ -21,9 +21,16 @@ Make sure you are using the lastest version of Raspbian:
...
+
+    Python 3:
+    $ sudo pip3 pifacecommon
```
Comment (tompreston) [inline README.md:30]: I think this should be `sudo pip3 install pifacecommon`

**Comment by psytester** [comment]
You are right, twice. I have changed the file.

**Comment by tompreston** [comment]
Looks great, thanks!

**Pull request merged**